clang/lib/CodeGen/TargetInfo.cpp

```diff
@@ -145,6 +145,23 @@ static bool occupiesMoreThan(CodeGenTypes &cgt,
   return (intCount + fpCount > maxAllRegisters);
 }
 
+static void addAMDGCOrNNVVMMetadata(const char *AnnotationName,
```

Comment:
Please add a comment about the function.

Reply:
Done in: 131f3f6c0d98a69a1ad7f6c99b7c31b0aeccb2f6

Reply:
Thanks.